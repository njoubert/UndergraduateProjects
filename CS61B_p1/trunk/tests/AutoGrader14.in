Erik does not live in the threadbare house.
Chuck lives around here.
Alex is the apothecary.
Bryan lives in the tan house.
The dentist lives around here.
The grocer lives in the threadbare house.
There is a orange house.
Dan lives in the fuchsia house.
The endodontist lives in the fieldstone house.
Bryan is not the banker.
What does Bryan do?
Who is the grocer?
What does Dan do?
Where does Alex live?
What does the occupant of the orange house do?
Who is the endodontist?

