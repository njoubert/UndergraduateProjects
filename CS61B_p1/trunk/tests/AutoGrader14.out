1. Erik does not live in the threadbare house.
2. Chuck lives around here.
3. Alex is the apothecary.
4. Bryan lives in the tan house.
5. The dentist lives around here.
6. The grocer lives in the threadbare house.
7. There is a orange house.
8. Dan lives in the fuchsia house.
9. The endodontist lives in the fieldstone house.
10. Bryan is not the banker.
Q: What does Bryan do?
A: Bryan is the dentist.
Q: Who is the grocer?
A: Chuck is the grocer.
Q: What does Dan do?
A: Dan is the banker.
Q: Where does Alex live?
A: Alex lives in the orange house.
Q: What does the occupant of the orange house do?
A: The apothecary lives in the orange house.
Q: Who is the endodontist?
A: Erik is the endodontist.
